packages/components/src/Button/index.ts

```diff
@@ -1 +1,2 @@
 export * from "./Button"
+export * from "./components/GenericButton"
```

Comment:
Does this mean we're now exporting the GenericButton? Makes sense for exporting the types but I wonder if it mean peeps will start trying to consume this / or if it was possible to access this before 🤔

Reply:
Doesn't look like `GenericButton` was available before, but this approach aligns with the new development pattern where we export internals to allow for custom builds.

If consumers actually need a custom button, then they should be able to create one. Not that the GenericButton is particularly good lol

Reply:
True, wasn't sure if we wanted to offer it but certainly not apposed 👍